dist/htmlminifier.js

```diff
@@ -285,8 +237,30 @@
       parseEndTag();
     }
 
-    function parseStartTag( tag, tagName, rest, unary ) {
-      var unarySlash = false;
+    function parseStartTag(input) {
+      var start = input.match(startTagOpen);
+      if (start) {
+        var match = {
```

Comment:
One option is to use Babel. Another option is to use Node (which supports a subset of ES6) and compile (deprecated?) browser version of minifier via browserify, which would — I hope — take care of ES6.